azure-pipelines.release.yml

```diff
@@ -38,3 +38,14 @@ steps:
   - script: |
       yarn publish:beachball -n $(npm.authtoken) -y
     displayName: 'beachball publish'
+
+  - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+    displayName: 📒 Generate Manifest
+    inputs:
+      BuildDropPath: $(System.DefaultWorkingDirectory)
+
+   - task: PublishPipelineArtifact@1
+    displayName: 📒 Publish Manifest
+    inputs:
+      artifactName: SBom-$(System.JobAttempt)
+      targetPath: $(System.DefaultWorkingDirectory)/_manifest
```

Comment:
Actually shouldn't this come before publish? I saw some discussion about whether this output should actually be published or not, but I do remember the docs saying after build and before publish.

Reply:
As long as it is associated with the build and can be retrieved it is okay. It was not a requirement to add it to each npm package (whcih is what we would have to do to embed it). 

We'd have to itterate over all packages and run the manifest tool.
Note that these manifests can be very big I've seen ones > 100Mb So we'd be bloating the npm packages...

Reply:
And since technically in this repo beachball does the version bumping after the build step, we would be getting the hashes of the files with previous versions and README's that would be out of data...

Reply:
Is there a requirement for pipeline artifact retention?